validation/raw_values.rst

```diff
@@ -62,9 +79,25 @@ Validation of arrays is possible using the ``Collection`` constraint::
         'gender' => new Assert\Choice(array(3, 4)),
         'file' => new Assert\File(),
         'password' => new Assert\Length(array('min' => 60)),
+        'tags' => new Assert\Required(array(
```

Comment:
I would make it optional instead, since required is the default and we don't explicit it on any other field of the example yet.